plugins/pytorch/CMakeLists.txt

```diff
@@ -75,3 +75,19 @@ if( VIAME_ENABLE_LEARN )
   add_subdirectory( cutler )
 
 endif()
+
+if( VIAME_ENABLE_REMAX )
```

Comment:
Where is this option defined? (with a cmake 'option()' command)

If this depends on the learn code, I would just use the VIAME_ENABLE_LEARN flag. If it adds few new dependencies and just depends on pytorch + some new dependencies, then a VIAME_ENABLE_PYTORCH-REMAX advanced flag would probably be best (see root cmake for other pytorch libs and how VIAME_ENABLE_PYTORCH-[SOMETHING] are defined, and also cmake/add_project_pytorch.cmake for how new packages are added). If no new dependencies then probably just use the pytorch flag as opposed to a new one.

Reply:
I moved the remax file based on cascade rcnn to the VIAME_ENABLE_LEARN file since it depends on that model. for the other two I added the flag to the highest level CMakeLists.txt file

Reply:
I would argue again that there shouldn't be a VIAME_ENABLE_REMAX flag, either only a VIAME_ENABLE_PYTORCH-REMAX or not another new ENABLE_X root flag and just re-using the ENABLE_LEARN for this

Reply:
Okay I'm changing to just use the ENABLE_LEARN flag. There is a use case in which a user wants to use a DINO remax model that they wouldn't need to have LEARN stuff, but that is more a one off example.